jobs/config.json

```diff
@@ -1693,15 +1696,18 @@
   "scenario": "kubernetes_e2e",
   "args": [
     "--env-file=platforms/gke.env",
-    "--env-file=jobs/ci-kubernetes-e2e-gke-large-deploy.env"
+    "--env-file=jobs/ci-kubernetes-e2e-gke-large-deploy.env",
+    "--down=false"
```

Comment:
oh, add --cluster=gke-large-deploy here and remove E2E_NAME env, as I forgot in kubemark. Same for rest of jobs.